### PR TITLE
Docker graphite api

### PR DESCRIPTION
**Pull request by goru97**
Adding graphite-api to docker-compose. 
This PR includes fixes done in https://github.com/rackerlabs/blueflood/pull/695

### CODE REVIEW

**Comment by goru97** [comment]
@shintasmith Take a look!

File: contrib/blueflood-docker/docker-compose.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 blueflood:
-  build: .
+  image: goru97/blueflood
```
Comment (shintasmith) [inline contrib/blueflood-docker/docker-compose.yml]: This has your name on it. Is it a path to github? Should we point to rackerlabs or a generic one?

File: contrib/blueflood-docker/docker-compose.yml [hunk 2]
```diff
@@ -20,4 +24,16 @@ cassandran:
...
 elasticsearch:
   image: elasticsearch:1.7
-  command: elasticsearch -Des.cluster.name="blueflood"
\ No newline at end of file
+  command: elasticsearch -Des.cluster.name="blueflood"
+
+#Graphite-API Integration
+
+graphite-api:
+  image: goru97/graphite-api
```
Comment (shintasmith) [inline contrib/blueflood-docker/docker-compose.yml]: Same here

File: contrib/blueflood-docker/docker-compose.yml (hunk 1 above)
Comment (goru97) [inline contrib/blueflood-docker/docker-compose.yml]: I can host the image on rackerlabs. @stackedsax do you have credentials for this account https://hub.docker.com/u/rackerlabs/ ?

**Comment by goru97** [comment]
@shintasmith you can give a conditional +1. We can use my account till it is hosted under rackerlabs.

**Comment by shintasmith** [comment]
@goru97 , if it doesn't take long, let's get this right the first time.

**Comment by usnavi** [comment]
@goru97 - we don't do conditional +1's.  Last time we did, https://github.com/rackerlabs/blueflood/pull/623, someone said that they'd come back and fix it....and I'm still waiting :-P

**Comment by goru97** [comment]
Closing in favor of https://github.com/rackerlabs/blueflood/pull/715

**Pull request closed without merging**